index.html

```
@@ -1493,6 +1610,12 @@ <h3 class="font-bold mb-2">Join Discord</h3>
         <div class="gh-stat-item text-blue-400"><span class="material-symbols-outlined text-sm">history</span> <span id="mActivity">Moderate</span></div>
       </div>
 
+      <div class="section-title">Competitive Intelligence</div>
+      <div id="competitive-dashboard-container" class="mb-8">
+        </div>
+      <div class="section-title">Description</div>
+      <p id="mDesc" class="text-sm text-zinc-600 leading-relaxed mb-6"></p>
+
       <div class="section-title">Description</div>
       <p id="mDesc" class="text-sm text-zinc-600 leading-relaxed mb-6"></p>
```

HTMLHint (1.9.2)

[error] The id value [ mDesc ] must be unique.

(id-unique)